server-utils/src/reactor.rs

```diff
@@ -148,3 +148,15 @@ impl RpcEventLoop {
 			});
 	}
 }
+
+#[cfg(test)]
+mod tests {
+	use super::*;
+
+	fn send_and_sync<T: Send + Sync>(_: T) {}
```

Comment:
I would have actually write this without args, and also I would have put it in the test itself as it is not useful to reuse
```suggestion
	fn send_and_sync<T: Send + Sync>() {}
```